### PR TITLE
Update and reformat the README

### PR DESCRIPTION
**Pull request by UCLGuichard**
Better formatting and updated information for the release.

### CODE REVIEW

**Comment by UCLGuichard** [comment]
@Luke-Pratley @dpshelio could you please quickly review ?

File: README.md [hunk 1]
```diff
@@ -1,78 +1,63 @@
...
+`C++` minimal dependencies:
 
-### C++ pre-requisites and dependencies
-
-- [CMake](http://www.cmake.org/): a free software that allows cross-platform compilation
-- [tiff](http://www.libtiff.org/): Tag Image File Format library
-- [OpenMP](http://openmp.org/wp/): Optional. Speeds up some of the operations.
-- [UCL/GreatCMakeCookOff](https://github.com/UCL/GreatCMakeCookOff): Collection of cmake recipes.
+- [CMake](http://www.cmake.org/) v3.9.2 A free software that allows cross-platform compilation
+- [GCC](https://gcc.gnu.org) v7.3.0 GNU compiler for `C++`
```
Comment (Luke-Pratley) [inline README.md]: This is the version needed for purify. I expect that sopt will work with much less.

File: README.md [hunk 2]
```diff
@@ -82,48 +67,94 @@ cmake -DCMAKE_BUILD_TYPE=Release ..
...
+* [Rafael E. Carrillo](https://www.researchgate.net/profile/Rafael_Carrillo2)
+* [Jason D. McEwen](http://www.jasonmcewen.org)
+* [Yves Wiaux](http://basp.eps.hw.ac.uk)
+* [Vijay Kartik](https://people.epfl.ch/vijay.kartik)
+* [Mayeul d'Avezac](https://github.com/mdavezac)
+* [Luke Pratley](https://about.me/luke.pratley)
+* [David Perez-Suarez](https://dpshelio.github.io)
+* [Ilektra Christidi](https://github.com/ilectra)
+* [Roland Guichard](https://github.com/UCLGuichard)
+
```
Comment (Luke-Pratley) [inline README.md]: I think Jason said to put the authors in alphabetical order?

File: README.md (hunk 1 above)
Comment (UCLGuichard) [reply README.md]: That's what is implemented in Travis. But I guess you won't use a different compiler version for purify and sopt ? That would be vicious.

File: README.md (hunk 2 above)
Comment (UCLGuichard) [reply README.md]: Yes, for the next round of reformatting.

**Pull request merged**